src/content/blog/2025-03-25-tus-node-server-v2.0.0.md

```diff
@@ -0,0 +1,191 @@
+author: murderlon
+date: 2025-03-25
+---
+
+The Node.js tus server can now run in all meta-frameworks and Node.js compatible
+runtimes. Deprecated options have been removed, and the server is ESM-only. The
+packages now require Node.js >=20.19.0, the release that backported
+`require(esm)`, allowing us to output ESM and import ESM packages without
+affecting consumers on CommonJS. But that’s not all, so let’s dive in.
+
+## What is tus?
+
+tus is an HTTP-based protocol for resumable file uploads. Resumable means that
+an upload can be picked up again at any moment, after it was interrupted for
+whatever reason, without the need to re-upload the previous data. An
+interruption may happen on purpose, if the user wants to pause, or by accident
+in case of a network issue or server outage (or your cat deciding to take a nap
+on the keyboard).
+
+The developers behind tus are currently working together with the HTTP working
+group inside the Internet Engineering Task Force (IETF) – the internet
+standards-setting organization that defines the network protocols powering the
+entire internet – to
+[make resumable uploads an official standard](https://tus.io/blog/2023/08/09/resumable-uploads-ietf)
+across the web.
+
+[tus-node-server][] is an official implementation of the tus resumable upload
+protocol, although it does not support the draft yet. It is capable of accepting uploads
+of all sorts and sizes, and storing them locally on disk, or remotely on Google
+Cloud Storage or AWS S3 (or any other S3-compatible storage system). Due to its
+modularization and extensibility, support for nearly any other cloud provider
+could easily be added.
+
+## Looking back
+
+tus Node.js [1.0.0](https://tus.io/blog/2023/09/04/tus-node-server-v100) brought
+modular packages, TypeScript, and stability. Since that major release, we’ve
+been shipping lots of improvements in minor versions, such as:
+
+- Support for [custom lockers][locker] and distributed locks. This prevents
+  concurrent access to the same resources, allowing for horizontal scaling
+  without sticky sessions.
+- Configuring a [max size][]. This makes it possible to limit the maximum upload
+  size and could also help to prevent large payload attacks.
+- Supporting more [protocol extensions][extensions].
+- A new [`@tus/azure-store`][] package for
+  [Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs).
+- Support for multi-tenant environments.
+- Lots of stability improvements to [`@tus/s3-store`][].
+- Allowing hooks to override metadata.
+- Customizing upload URLs and
+  [custom nested directories](https://github.com/tus/tus-node-server/tree/main/packages/server#example-store-files-in-custom-nested-directories).
+- [And much more...](https://github.com/tus/tus-node-server/compare/%40tus/server%401.0.0...%40tus/server%402.0.0)
+
+We did all this, while maintaining backwards compatibility. It has been battle tested at
```

Comment:
Can you specify what 'it' is, in the second sentence on this line? I wasn't 100% from the context to what you are referring.